tidalapi/session.py

```diff
@@ -633,12 +633,12 @@ def save_session_to_file(self, session_file: Path):
                 "is_pkce": {"data": self.is_pkce},
                 # "expiry_time": {"data": self.expiry_time},
             }
-            with session_file.open("w") as outfile:
+            with open(session_file, "w") as outfile:
```

Comment:
session_file is a Path object, what's the point of reverting it to the plain `open()`?

If anything, rest of the code should be converted to Path.

Reply:
> If anything, rest of the code should be converted to Path.

Agreed